includes/class-webmention-admin.php

```diff
@@ -155,13 +161,26 @@ public static function comment_columns( $columns ) {
 	 * Add admin menu entry
 	 */
 	public static function admin_menu() {
-		add_options_page(
-			'Webmention',
-			'Webmention',
-			'manage_options',
-			'webmention',
-			array( 'Webmention_Admin', 'settings_page' )
-		);
+		$title = __( 'Webmention', 'webmention' );
+		// If the IndieWeb Plugin is installed use its menu.
+		if ( class_exists( 'IndieWeb_Plugin' ) ) {
```

Comment:
thinking of usability... shouldn't we always add both? I created a settings page, because this was feedbacked a lot of times... I am not sure if everyone searches in the IndieWeb area!?! what do you think?

perhaps @chrisaldrich can help here?